tests/test_dem.py

```diff
@@ -10,14 +10,8 @@
 import numpy as np
 from xdem.dem import DEM
 
-path_module = os.path.dirname(os.path.dirname(os.path.abspath(inspect.getsourcefile(xdem))))
-
-#TODO: move datasets to a "dataset" folder like in geoutils
-EXAMPLE_PATHS = {
-    "dem1": "examples/Longyearbyen/data/DEM_2009_ref.tif",
-    "dem2": "examples/Longyearbyen/data/DEM_1995.tif",
-    "glacier_mask": "examples/Longyearbyen/data/glacier_mask/CryoClim_GAO_SJ_1990.shp"
-}
+xdem.examples.download_longyearbyen_data(overwrite=False)
+xdem.examples.FILEPATHS["longyearbyen_ref_dem"]  # This is the same as EXAMPLE_PATHS["dem1"] before
```

Comment:
This line is not necessary! I just wanted to show how one would use it!

Reply:
Ups! went a bit fast :D